### ISSUE TITLE
childProcess.spawn escaping quotes on windows

### ISSUE THREAD

**Issue by zzzzBov** [kimmobrunfeldt/chokidar-cli#61]
when a command such as `chokidar *.* --initial -c "echo ""Hello World!"""` is executed on windows, the output is

```
\"Hello World!\"
```

rather than the expected

```
"Hello World!"
```

[This appears to be caused by some weird default behavior of `childProcess.spawn` when `windowsVerbatimArguments` is not set to `true`][1].

The quick fix is to just add the option to [utils.js][2]

```js
child = childProcess.spawn(SHELL_PATH, [EXECUTE_OPTION, cmd], {
  cwd: opts.cwd,
  stdio: opts.pipe ? 'inherit' : null,
  windowsVerbatimArguments: true
});
```

however `child_process.spawn` now has a `shell` option that looks like it would simplify things here:

```
child = childProcess.spawn(cmd, {
  cwd: opts.cwd,
  stdio: opts.pipe ? 'inherit' : null,
  shell: true
})
```

[1]: https://github.com/nodejs/node/issues/5060#issuecomment-179587557
[2]: https://github.com/kimmobrunfeldt/chokidar-cli/blob/master/utils.js#L35

**Comment by XhmikosR** [comment]
Yup, I was actually experimenting with this myself because otherwise tests do fail on Windows (with git bash). See https://github.com/kimmobrunfeldt/chokidar-cli/compare/master...XhmikosR:test-win

Which would also fix #62.

But then I couldn't make tests pass without setting `pipe: true` in tests.

We need to add cross-platform CI testing for sure, and look into switching to `shell` ASAP.
